### PR TITLE
migrate playground to webpack 4

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: playground/package.json [hunk 1]
```diff
@@ -1,60 +1,48 @@
...
+    "start": "webpack-dev-server --mode development --open",
+    "dev": "webpack --mode development",
+    "build": "rm -rf dist && webpack --mode production --optimize-minimize",
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "Sebastian Benz",
-  "license": "Apache-2.0",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (kristoferbaxter) [inline playground/package.json]: Is there a reason for the changes here?

1. License to ISC
2. Removal of description
3. Name change `playground` -> `playground2`
4. From pinned dependencies to hat dependencies
5. Removal of tests and linting.

**Comment by sebastianbenz** [comment]
Thanks everyone for already looking at this! It's still work in progress. Sorry for not making this explicit.

File: playground/package.json (hunk 1 above)
Comment (sebastianbenz) [reply playground/package.json]: see below :-)

**Comment by sebastianbenz** [comment]
PTAL - sorry about the eslint related noise.

**Comment by kristoferbaxter** [comment]
No worries! Thanks for the update.

File: playground/package.json [hunk 2]
```diff
@@ -1,60 +1,53 @@
...
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "An interactive AMP playground.",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --config webpack.common.js",
-    "build": "rm -rf dist && webpack --config webpack.prod.js && cp -R static/* dist",
+    "start": "webpack-dev-server --mode development --open",
+    "dev": "webpack --mode development",
+    "build": "rm -rf dist && webpack --mode production --optimize-minimize && cp -R static/* dist",
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply playground/package.json]: Hah! Thanks for making me look into this again. CopyWebpackPlugin is exactly what I needed (and what wasn't released yet when I looked into it). Done.

File: playground/package.json [hunk 3]
```diff
@@ -1,60 +1,53 @@
...
-    "standard": "10.0.3",
-    "style-ext-html-webpack-plugin": "4.0.1",
-    "style-loader": "0.21.0",
-    "url-loader": "0.5.9",
-    "url-search-params": "0.9.0",
-    "wade": "0.3.3",
-    "webpack": "3.11.0",
-    "webpack-dev-server": "2.11.2",
-    "webpack-merge": "4.1.2"
+    "@babel/core": "^7.1.2",
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply playground/package.json]: Just me not being able to use NPM properly...

[the remark this answers is not in the document]
Comment (sebastianbenz) [reply playground/package.json]: done

File: playground/src/app.js [hunk 4]
```diff
@@ -168,15 +169,7 @@ Button.from(document.getElementById('show-menu'), () => {
...
-  import('./formatting/base.js')
-    .then(formatHtml => {
-      const formattedCode = formatHtml.default(editor.getSource());
-      editor.setSource(formattedCode);
-    })
-    .catch(e => {
-      console.error(e);
-      snackbar.show('Could not fetch formatter');
-    });
+  formatter.format(editor.getSource()).then(formattedCode => editor.setSource(formattedCode));
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply playground/src/app.js]: Absolutely, I didn't use it for consistency as it's nowhere else used.

**Comment by sebastianbenz** [comment]
PTAL

File: playground/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (kristoferbaxter) [reply playground/package.json]: Awesome, glad to see it worked for you!

**Comment by sebastianbenz** [comment]
Thanks for the review

**Pull request merged**